specs/Standards/Tokens/FungibleTokenCore.md

```diff
@@ -0,0 +1,356 @@
+5. `alice` calls `dex::deposit({"token": "wltc", "amount": "100000000000000"})`.
+   1. `dex` calls `wltc::transfer_from({"owner_id": "alice", "new_owner_id": "dex", "amount": "100000000000000"})`
+6. `alice` calls `dex::trade({"have": "wltc", "have_amount": "900100000000", "want": "wbtc", "want_amount": "8000000000"})`.
+   - `dex` calls `wbtc::transfer({"new_owner_id": "alice", "amount": "8000000000"})`
+   - `dex` calls `wltc::transfer({"new_owner_id": "bob", "amount": "900100000000"})`
+
+<hr/>
+</details>
+
+**High-level explanation**
+
+Let's first set up an open order by Bob on the DEX. It's similar to `Token deposit to a contract` example above.
+- Bob sets an allowance on wBTC to DEX
+- Bob calls deposit on Dex for wBTC.
+- Bob calls DEX to make an new sell order.
+
+Then Alice comes and decides to fulfill the order by selling his wLTC to Bob on DEX.
+Alice calls the DEX
+- Alice sets the allowance on wLTC to DEX
+- Bob calls deposit on Dex for wLTC.
+- Then calls DEX to take the order from Bob.
+
+When called, DEX makes 2 async transfers calls to exchange corresponding tokens.
+- DEX calls wLTC to transfer tokens DEX to Bob.
+- DEX calls wBTC to transfer tokens DEX to Alice.
+
+**Technical calls**
+
+1. `bob` calls `wbtc::ft_transfer_call({"receiver_id": "dex", "amount": "8000000000", "msg": "deposit"})`.
+   1. makes async call `dex::ft_on_transfer({"sender_id": "bob", "amount": "8000000000", "msg": "deposit"})`.
+   2. attaches a callback `wbtc::ft_resolve_transfer({"sender_id": "bob", "receiver_id": "dex", "amount": "8000000000"})`.
+   3. the deposit completes on `dex`, all fungible tokens are used, so it returns `0` as the number of unused tokens back to `wbtc`.
+   4. the `wbtc::ft_resolve_transfer` function receives success as the execution outcome and `0` as the value. This means 8000000000 will be subtracted from `bob`'s balance.
+2. `bob` calls `dex::trade({"have": "wbtc", "have_amount": "8000000000", "want": "wltc", "want_amount": "900100000000"})`.
+3. `alice` calls `wltc::ft_transfer_call({"receiver_id": "dex", "amount": "100000000000000", "msg": "deposit"})`.
+   1. makes async call `dex::ft_on_transfer({"sender_id": "alice", "amount": "100000000000000", "msg": "deposit"})`.
+   2. attaches a callback `wltc::ft_resolve_transfer({"sender_id": "alice", "receiver_id": "dex", "amount": "100000000000000"})`.
+   3. the deposit completes on `dex`, all fungible tokens are used, so it returns `0` as the number of unused tokens back to `wltc`.
+   4. the `wltc::ft_resolve_transfer` function receives success as the execution outcome and `0` as the value. This means 100000000000000 will be subtracted from `bob`'s balance.   
+4. `alice` calls `dex::trade({"have": "wltc", "have_amount": "900100000000", "want": "wbtc", "want_amount": "8000000000"})`.
+   - `dex` calls `wbtc::transfer({"new_owner_id": "alice", "amount": "8000000000"})`
+   - `dex` calls `wltc::transfer({"new_owner_id": "bob", "amount": "900100000000"})`
```

Comment:
Let's remove this example, since it talks about NEP-21

Reply:
Removed in 42d1b8f1d7f1ae4f64e18706ee79ae0a0c5f8ade